unit-tests/src/test/scala/scala/scalanative/unsafe/CStringTest.scala

```diff
@@ -121,13 +120,13 @@ object CStringSuite extends tests.Suite {
 
       val jstr2: String = fromCString(arr.at(0))
 
-      assertEquals(strcmp(cstr1, cstr2), 0)
-      assertEquals(strlen(arr.at(0)), 10)
+      assertEquals(0, strcmp(cstr1, cstr2))
+      assertEquals(10, strlen(arr.at(0)))
 
       assertEquals(jstr2, jstr1 * 2)
       assertEquals(jstr2.last, 'd')
-      assertEquals(!(cstr1 + 4), 'd')
```

Comment:
I believe these two tests were using pointer arithmetic and now they are using the index method.

Reply:
Thank you for the review.  My brain must be stuck in C land, where the two are exactly equivalent.  
By my understanding of Scala, they two ought to be equivalent in results (I am avoiding the word 'functionally'),
A CString is an Array[Byte]. One ought to be able to index off it.

I can restore the pointer math, if you think there is even the slightest chance that I introduced an error.
I thought that an Array index would be more natural to a Scala reviewer.  

Please advise.  My goal is to make people's work easier, not harder.

L

Reply:
You could leave the test as is and add 2 new cases? When I was porting I didn't change anything for the most part. `assert -> assertTrue` for example. You changed to `assertEquals` which I think is okay but it widens the diff.

Reply:
IIUC, you are suggesting adding two cases using pointer math?  I can do that.

Based on your "widening" concerns and the cadence you and Sébastien have established
I am going to convert this to WIP and do the AssertEquals in a subsequent PR, once the
dust has settled.

Reply:
I think the widening may be fine for one test but I would leave the pointer arithmetic test as is or add 2 new cases. I am not an expert but `assertEquals` is probably a big improvement.